src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2020 by 티거, 제이미
+ * All rights reserved.
+ *
+ * @author      티거, 제이미
+ * @version     1.0
+ * @date        11 Feb 2020
+ *
+ */
+
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    private final static String REGEX_CHECK_NOT_NUMBER = "[^0-9]";
+    private final static String DELIMITER = ",|:";
+    private final static String CUSTOM_DELIMITER_PATTERN = "//(.)\n(.*)";
+    private final static String HYPHEN = "-";
+
+    public static int splitAndSum(String input) {
+        if (isNullOrBlank(input)) {
+            return 0;
+        }
+        List<String> inputs = split(input);
+        return addAll(inputs);
+    }
+
+    private static boolean isNullOrBlank(String input) {
+        return input == null || input.equals("");
```

Comment:
```suggestion
        return input == null || input.isEmpty();
```

자바에서 기본적으로 제공하는 기능을 사용해보는 것은 어떨까요?
추가적으로 empty와 blank는 어떠한 차이가 있는지 고민해보아요.

Reply:
#### equals("")와 isEmpty()의 차이

1. 일단 명시적으로 차이가 있습니다.

  - isEmpty()가 조금 더 명확히 들어옵니다!

2. 실질적 차이

  - isEmpty의 경우엔 해당 객체(여기선 input)의 length가 0인지 확인하여 리턴합니다.
    > 아주 조금 더 빠르다고 하네요!

  - equals("")의 경우엔, 문자열 풀에 있는 ""와 비교를 하여 비교 결과를 리턴합니다.
    > 비교시에 Type과 Length체크 후 일치하면 문자열과 비교한다고 나와있었습니다.

짚어주셔서 감사합니다(_ _)

Reply:
"".isEmpty() == true
" ".isEmpty() == false
"".isBlank() == true
" ".isBlank() == true

Blank는 공백을 무시한다. 라고 생각하시면 좋을 것 같아요!
Java11 버전에선 isBlank() 라는 것을 제공하기 때문에 헷갈리지 않도록 피드백 드렸습니다 :)

Reply:
아하, 넵넵 말씀주셔서 감사합니다 :)